Lib/test/test_email/test_headerregistry.py

```diff
@@ -1643,10 +1643,10 @@ def test_fold_overlong_words_using_RFC2047(self):
         self.assertEqual(
             h.fold(policy=policy.default),
             'X-Report-Abuse: =?utf-8?q?=3Chttps=3A//www=2Emailitapp=2E'
-                'com/report=5F?=\n'
-            ' =?utf-8?q?abuse=2Ephp=3Fmid=3Dxxx-xxx-xxxx'
-                'xxxxxxxxxxxxxxxxxxxx=3D=3D-xxx-?=\n'
-            ' =?utf-8?q?xx-xx=3E?=\n')
+                'com/report=5Fabuse?=\n'
```

Comment:
Expected result was modified because the previous algorithms tended not to use all available space in the initial encoded words.